packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -297,9 +274,7 @@ const ModalManager = () => {
 
     return (
         <React.Fragment>
-            {is_jurisdiction_modal_visible && <JurisdictionModal openPasswordModal={openRealPasswordModal} />}
```

Comment:
we can remove these store values as well `is_jurisdiction_modal_visible`, `is_cfd_verification_modal_visible`

Reply:
done